.github/workflows/gh-pages.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         run: git fetch --tags origin
 
       - name: Setup | mdBook | 1/2
-        uses: hecrj/setup-rust-action@master
+        uses: hecrj/setup-rust-action@8708beccd22540a3f955ae10cc884af27ca81bf5
```

Comment:
Same here, maybe just a single tracking issue to update all of these?

Reply:
See above.